README.md

```
@@ -1,20 +1,40 @@
+
 ## Quick start
 
-1. Edit `config.toml` and set `caldav_base`, `db_path`, and a strong `hmac_secret`.
-2. Build:
+1. Update `config.toml` values (`bind`, `caldav_base`, `worker_url`, `worker_secret`, `hmac_secret`).
+2. Build and run:
+   ```bash
+   docker compose up -d --build
+   ```
+3. Basic endpoint checks:
    ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)